doc/release-notes/rl-2511.section.md

```diff
@@ -9,6 +9,10 @@
   and newer series. However, embedded chips without LSX (Loongson SIMD eXtension), such as 2K0300 SoC, are not
   supported. `pkgsCross.loongarch64-linux-embedded` can be used to build software and systems for these platforms.
 
+- Darwin has switched to using the system libc++. This was done for improved compatibility and to avoid ODR violations.
+  If a newer C++ library feature is not available on the default deployment target, you will need to increase the deployment target or use libc++ from LLVM.
```

Comment:
Maybe shouldn’t recommend the LLVM `libc++` at present? And this is also probably zero impact for most users, so it might belong in the backward incompatibilities section instead.

Reply:
Yeah, might just keep the deployment target note and also update the Darwin documentation not to mention it.

Reply:
I updated the Darwin documentation to mention that you can use libc++ from LLVM, but it’s not recommended.